### ISSUE TITLE
SB PCI on Windows 98

### ISSUE THREAD

**Issue by bob12111** [joncampbell123/dosbox-x#2726]
When I installed Windows 98 on DOSBOX-X, I noticed the music in some of the games sounded different compared to when they were played on another machine with Win 98 installed via VM-Ware.  It appears that the latter uses SB PCI, is there a way to install that on DOSBOX-X?

**Comment by joncampbell123** [comment]
VM ware is emulating the SB PCI cards? Is it full em10k1 emulation, like the SB Live! PCI I used to have in the late 1990s?

**Comment by bob12111** [comment]
I'm not sure.  Try it for yourself...

**Comment by maron2000** [comment]
VMWare supported Creative Ensoniq Audio PCI ES1371, so I think its not em10k1 emulation.

**Comment by Wengier** [comment]
@maron2000 Yes, you are right VMware uses Ensoniq Audio PCI ES1371 by default (although it can be changed to SB16). ES1371 and SB Live! use similar but different DOS drivers, although IIRC the ES1371 DOS driver does not work in VMware for some reason, and you have to change to SB16 to work in DOS in such case.

**Comment by joncampbell123** [comment]
Not sure if I have any ES1371-based hardware, but I do have the datasheet I dug out of the ALSA project's archives some time ago.

http://hackipedia.org/browse.cgi/Computer/Platform/PC%2c%20IBM%20compatible/Sound%20and%20Music/Ensoniq/ENSONIQ%20AudioPCI%e2%84%a2%2097%20ES1371%20Specification%20%281997%2d10%2d01%29%2epdf
